### DISCUSSION TITLE
Powershell Commands

### DISCUSSION THREAD

**Discussion by olden-goldie** [valinet/ExplorerPatcher#4008] [Q&A]
I had found on this hub commands to allow EP to work properly after 11h24.  I believe they were commands to make changes in powershell but I am unable to find them again.  Can anyone help me with this?

**Comment by pyrates999** [comment]
they're here: https://github.com/valinet/ExplorerPatcher/releases

**Reply by olden-goldie** [reply]
Thank you!
